server/server.js

```diff
@@ -1,13 +1,22 @@
-const express = require('express');
 const path = require('path');
+const cacheMiddleware = require('apicache').middleware;
```

Comment:
While this works, the paradigm in this app is to put the middleware in the `app.js` file here: https://github.com/bocoup/aria-at-app/blob/main/server/app.js. 

My decision in this file to put `/aria-at` path in the `server.js` file was for the `express.static` functionality. For this,  though, I recommend doing as I said above.

Reply:
@evmiguel sure thing. could you clarify, if i move the middleware / path to the `app.js` file, the only thing in `server.js` file will be the `listen` right?

Reply:
@s3ththompson yes!